src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/fieldFilterTypes/SelectionFieldFilterType.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+import React from 'react';
+import {autorun, computed, toJS, untracked, when} from 'mobx';
+import equals from 'fast-deep-equal';
+import MultiSelectionStore from '../../../stores/MultiSelectionStore';
+import MultiAutoComplete from '../../MultiAutoComplete';
+import AbstractFieldFilterType from './AbstractFieldFilterType';
+import selectionFieldFilterTypeStyles from './selectionFieldFilterType.scss';
+
+class SelectionFieldFilterType extends AbstractFieldFilterType<?Array<string | number>> {
```

Comment:
Custom entity should not be a big deal, because you can pass the `resourceKey` in the list configuration.

Actually Thomas and I agreed that there is only one version, not having multiple ways of assigning stuff as in the `Selection` field type for forms, therefore I implemented it this way.

And even if we would build something similar, I would not use the `Selection` field type for it. We would have to translate all the properties in some way, also properties which are not type checked... Sounds to me like this could easily fail in some way.